tools/installpsh-osx.sh

```diff
@@ -93,14 +93,13 @@ if (( $EUID != 0 )); then
 fi
 
 #Check that sudo is available
-if [[ "$SUDO" -eq "sudo" ]]; then
-
-    $SUDO -v
-    if [ $? -ne 0]; then
-      echo "ERROR: You must either be root or be able to use sudo" >&2
-      exit 5
-    fi
-fi
+#if [[ "$SUDO" -eq "sudo" ]]; then
```

Comment:
We should remove this commented code if it's not needed

Reply:
It's not clear to me if this is needed for local execution of this script or if anyone actually uses these scripts locally outside of CI